test/gurka/test_instructions_roundabout_multicue.cc

```diff
@@ -0,0 +1,198 @@
+  static void SetUpTestSuite() {
+    constexpr double gridsize_metres = 100;
+
+    const std::string ascii_map = R"(
+            I      O
+            |      |
+            H      N    P
+           / \    / \   |
+        A-B   E--F   L--M
+           \ /    \ /
+            C      J
+            |      |
+            D      K
+    )";
+
+    const gurka::ways ways =
+        {{"AB",
+          {{"highway", "primary"},
+           {"name", "Main Street"},
+           {"ref", "A 1"},
+           {"maxspeed", "50 mph"},
+           {"destination", "München;Kürten;Dhünn"},
+           {"destination:ref", "A 95;B 2;B 300"}}},
+         {"BCEHB",
+          {{"highway", "primary"},
+           {"oneway", "yes"},
+           {"junction", "roundabout"},
+           {"maxspeed", "50 mph"},
+           {"name", ""}}},
+         {"CD", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "1st Avenue"}}},
+         {"EF",
+          {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "Main Street"}, {"ref", "A 1"}}},
+         {"HI", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", ""}}},
+         {"FJLNF",
+          {{"highway", "primary"},
+           {"oneway", "yes"},
+           {"junction", "roundabout"},
+           {"maxspeed", "50 mph"},
+           {"name", ""}}},
+         {"JK", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "2nd Avenue"}}},
+         {"LM",
+          {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "Main Street"}, {"ref", "A 1"}}},
+         {"NO", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", ""}}},
+         {"PM", {{"highway", "primary"}, {"maxspeed", "50 mph"}, {"name", "3rd Avenue"}}}};
+
+    const auto layout =
+        gurka::detail::map_to_coordinates(ascii_map, gridsize_metres, {5.1079374, 52.0887174});
+    map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_roundabout_multicue",
+                            {{"mjolnir.admin",
+                              {VALHALLA_SOURCE_DIR "test/data/netherlands_admin.sqlite"}}});
+  }
+};
+
+gurka::map InstructionsRoundaboutMulticue::map = {};
+
+/*************************************************************/
+
+TEST_F(InstructionsRoundaboutMulticue, StartThenEnterRoundaboutMulticueOrdinalOnly) {
+  auto result = gurka::route(map, "A", "I", "auto");
+  gurka::assert::raw::expect_maneuvers(result, {DirectionsLeg_Maneuver_Type_kStart,
+                                                DirectionsLeg_Maneuver_Type_kRoundaboutEnter,
+                                                DirectionsLeg_Maneuver_Type_kRoundaboutExit,
+                                                DirectionsLeg_Maneuver_Type_kDestination});
+  int maneuver_index = 0;
+
+  // Verify the start then enter roundabout multicue
+  gurka::assert::raw::expect_instructions_at_maneuver_index(
+      result, maneuver_index, "Drive east on Main Street/A 1.", "",
+      "Drive east on Main Street, A 1. Then Enter the roundabout and take the 3rd exit.",
```

Comment:
Should `Enter` be capitalized? like `Drive east on Main Street, A 1. Then enter the roundabout ...` ?